src/test/java/com/google/javascript/gents/multiTests/import_rewrite/export.ts

```diff
@@ -1,23 +1,17 @@
 export function B() {}
-
-export const x = 4;
-
-export const y = 8;
+obj.C.x = 4;
```

Comment:
this won't work as it doesn't define `obj.C`, does it? Or am I missing something?